pyccel/codegen/printing/pycode.py

```diff
@@ -78,7 +78,8 @@ def _print_FunctionDef(self, expr):
 
         decorators = expr.decorators
         if decorators:
-            for n,args in decorators.items():
```

Comment:
It looks like the original code was buggy. Was it working before?

Reply:
Yes, it was. Redbaron has a decorator object which meant we could treat it separately, but python ast stores decorators as function calls, hence the need to change the way we manage this loop